HDF5Examples/JAVA/README-MAVEN.md

```diff
@@ -0,0 +1,233 @@
+# HDF5 Java Examples Maven Integration
+
+This directory contains 62 Java examples demonstrating the usage of HDF5 Java bindings, organized into categories and deployable as a Maven artifact.
```

Comment:
Probably best to remove all references to a specific number of examples in this file, as it's maintenance burden for no reason

Reply:
Fixed